added package hex sticker

ambertin opened the pull request:
Made a hex sticker for the package and added it to the README! Should now show up on the front of the GitHub page when people come to view it! Yay!

beanumber reviewed (changes requested):
Can you please upload the code to make the sticker to `data-raw`?

ambertin commented:
Went ahead and added the code and files for generating the hex sticker! Also it seems as though there was a typo in what you added to the RBuildignore (for ignoring the install proj packages files) that was causing the project build (and therefore Travis) to fail. Corrected that as well.
